src/components/ProjectPage/ProjectPage.js

```diff
@@ -136,9 +136,9 @@ class ProjectPage extends PureComponent<Props> {
             <Fragment>
               <Spacer size={30} />
               <Paragraph>
-                Please install the dependencies so you can use the scripts that
-                are needed by your project. This is OK and happens if you're
-                freshly cloning a project from Github.
+                Please install dependencies so you can use the scripts that are
```

Comment:
Thanks for the new copy. That's better 👍 I'll update it

Reply:
OK, I added your copy. I also changed the styling a bit.
I'm also using enum-style conditional rendering instead of `&&` conditionals in `ProjectPage`. I think we have used rendering methods before but I like that's less code. 
What do you think is it OK to use?

Reply:
Great new look, I love it with the icon 😄 

Enum rendering is interesting, I don't think I've come across it before. LGTM, since it's just a code style thing I think it's fine to use, can always be refactored later if necessary. But yeah I do agree it seems cleaner.